src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApiService.java

```diff
@@ -212,5 +215,20 @@ public PrisonAppointment getPrisonAppointment(long appointmentId) {
                 .blockOptional()
                 .orElse(null);
     }
+
+    public void deleteAppointment(final Long appointmentId) {
+        try {
+            webClient.delete()
+                    .uri("/appointments/{appointmentId}", appointmentId)
+                    .retrieve()
+                    .bodyToMono(String.class)
+                    .block();
+        } catch (WebClientResponseException e) {
+            if (!e.getStatusCode().equals(NOT_FOUND)) {
+                throw e;
```

Comment:
Throwing an exception here will stop the delete and yet you could carry on, delete any remaining appointments in prison-api and then delete the VideoLinkBooking in whereabouts-api.

Reply:
This is ignoring 404s and only stopping on 500s - that makes sense right? If you can't delete the first appointment in nomis it would be better to error and let the user retry

Reply:
removed try catch. Replaced with onResumeError(...